upgrade_test.py

```diff
@@ -1000,18 +1000,13 @@ def test_generic_cluster_upgrade(self):
         # generate random order to upgrade
         nodes_to_upgrade = self.shuffle_nodes_and_alternate_dcs(list(self.db_cluster.nodes))
 
-        # Upgrade all nodes that should be rollback later
-        upgraded_nodes = []
-        for node_to_upgrade in nodes_to_upgrade[:self.params.get('num_nodes_to_rollback')]:
-            self._start_and_wait_for_node_upgrade(node_to_upgrade, step=next(step))
-            upgraded_nodes.append(node_to_upgrade)
-
-        # Rollback all nodes that where upgraded (not necessarily in the same order)
-        random.shuffle(upgraded_nodes)
-        InfoEvent(message='Upgraded Nodes to be rollback are: %s' %
-                  ", ".join(node.name for node in upgraded_nodes)).publish()
-        for node in upgraded_nodes:
-            self._start_and_wait_for_node_rollback(node, step=next(step))
+        # Upgrade a node that should be rollback later
+        node_to_upgrade = random.choice(nodes_to_upgrade)
+        self._start_and_wait_for_node_upgrade(node_to_upgrade, step=next(step))
+
+        # Rollback the node that was upgraded
+        InfoEvent(message='Upgraded Node to be rollback is: %s' % node_to_upgrade).publish()
```

Comment:
why change the code, and not change `num_nodes_to_rollback: 1` ?

Reply:
my mistake, corrected